action.yml

```diff
@@ -29,14 +33,18 @@ runs:
             echo "Assigning issue $ISSUE_NUMBER to $LOGIN"
             echo "Using the link: https://api.github.com/repos/$REPO/issues/$ISSUE_NUMBER/assignees"
             curl -H "Authorization: token $GITHUB_TOKEN" -d '{"assignees":["'"$LOGIN"'"]}' https://api.github.com/repos/$REPO/issues/$ISSUE_NUMBER/assignees
+            echo "What is the input message?"
             if [[ ! -z $INPUT_MESSAGE ]]; then
-              jq -n -r --arg body "$INPUT_MESSAGE" '{body: $body}' > payload.json
+              jq -n -r --arg body "$ISSUE_CURRENTLY_ASSIGNED_MESSAGE" '{body: $body}' > payload.json
               curl -X POST -H "Authorization: token $GITHUB_TOKEN" --data @payload.json https://api.github.com/repos/$REPO/issues/$ISSUE_NUMBER/comments
             fi
           else
```

Comment:
May want to check whether $ISSUE_CURRENTLY_ASSIGNED_MESSAGE is blank here.

Reply:
okay i added an if statement like the one that exists for input message..I hope that's doing the right thing. let me know if not!!

Reply:
I’d just suggest rechecking the action by testing it in one of your repos and pointing the workflow file to your fork/branch.

try it with workflow file specifying blank input (edge case now but whatevs) and making sure it stays quiet.